lib/python-beta/src/bailo/helper/model.py

```diff
@@ -0,0 +1,105 @@
+from __future__ import annotations
+
+import os
+from typing import Any
+
+from bailo.core import Client, ModelVisibility
+from bailo.helper.utils import retrieve_model
+
+
+class Model:
+    def __init__(
+        self,
+        client: Client,
+        name: str,
+        description: str,
+        visibility: ModelVisibility | None = ModelVisibility.Public,
```

Comment:
You're encoding backend logic in here.  Just use `visibility: ModelVisibility` here, as the backend will always return it.